releasenotes/notes/update-cephadm-collection-f269d7cbff39f0de.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Updates Cephadm Ansible collection to the latest 1.21.0 version. Refer to
+    release notes for the list of changes:
+    https://github.com/stackhpc/ansible-collection-cephadm/releases
```

Comment:
nit: Make it a link?

Reply:
there's no need https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links